website/blog/2024-02-08-kubernetes-frontends/index.mdx

```diff
@@ -0,0 +1,195 @@
+---
+slug: kubernetes-frontends
+title: The inner workings of kubernetes management frontends - A software engineers' perspective
+authors: [ christophenne ]
+tags: [ glasskube, kubernetes, frontends ]
+---
+
+# The inner workings of kubernetes management frontends - A software engineers' perspective
```

Comment:
About the title: I think it's good but maybe a little long? But I don't really have a good suggestion how to make it shorter, so … :shrug: 

Additionally:
1. I would change it to title case (e.g. https://titlecaseconverter.com/) (personal preference)
2. This is a pet peeve of mine: The "-" should be replaced with an emdash: https://www.thesaurus.com/e/grammar/em-dash/ (also in some other places in the article, I will not comment on those)